### ISSUE TITLE
Params.txt

### ISSUE THREAD

**Issue by ghost** [gamelodge/work-test#2]
In your Read-me you mention that the login details will be there, the file I have when forked from you is empty?

**Comment by gamelodge** [comment]
Yes :+1:  That is correct -- Well done for being the first one to show they can use the issue que. becasue we work with git & git hub its important for applicants to be able to use git or quickly learn how to use it. -- I will email you the details that you need.
